packages/lab/src/Table/stories/TableColumnRenderers.stories.mdx

```diff
@@ -0,0 +1,202 @@
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+- `format: String`: the format used to format the value the default value is `ISO8601`.
+
+output:
+- `col: HvTableColumnConfig`: The modified column configuration with the parameters required to render the text.
+
+<Preview withToolbar>
+  <Story inline story={DateColumnRenderer} />
+</Preview>
+
+## Expand column renderer
+
+This functions in conjunction with the `useHvRowExpand` hook renders a column with a button to open additional content in the table.
+The column generated will be *right* aligned and have the same properties as the `hvTextColumn` column.
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+- `expandRowButtonAriaLabel: String`: The aria label for the button when the expander is closed.
+- `collapseRowButtonAriaLabel: String`: The aria label for the button when the expander is open.
+- `getCanRowExpand: (row: HvRowInstance) => boolean`: The function executed to determine whether the expander button is able to be opened.
+
+output:
+- `col: HvTableColumnConfig`: The modified column configuration with the parameters required to render the text.
+
+<Preview withToolbar>
+  <Story inline story={ExpandColumnRenderer} />
+</Preview>
+
+## Switch column renderer
+
+This functions renders a column with a switch component embedded that follows the state of a `boolean` value.
+The column generated will be *right* aligned.
+To change the value of the switch the data in the table must be changed.
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+- `switchLabel: String`: The aria label for the switch.
+- `falseLabel: String`: The label for the left side of the switch.
+- `trueLabel: String`: The label for the right side of the switch.
+- `switchProps: HvBaseSwitchProps`: extra props passed to switch, these values correspond to the `HvBaseSwitch` props.
+
+output:
+- `col: HvTableColumnConfig`: The modified column configuration with the parameters required to render the text.
+
+<Preview withToolbar>
+  <Story inline story={SwitchColumnRenderer} />
+</Preview>
+
+## Tag column renderer
+
+This functions renders a column with a tag component embedded that follows the state of a `json` value.
+The column generated will be *right* aligned.
+To change the value color and styles of the tag the data in the table must be changed.
+The function receives 3 key values that correspond to properties present in the table cell value.
+
+parameters:
+- `col: HvTableColumnConfig`: The column configuration to modify.
+- `valueDataKey: String`: The aria label for the switch.
+- `colorDataKey: String`: The label for the left side of the switch.
+- `textColorDataKey: String`: The label for the right side of the switch.
+- `fromRowData: Boolean`: if `false` uses the raw value of the table coming from the data.
```

Comment:
I think we need a parameter to control the tag properties, and that it should default to semantic, instead of categorical.